app/app.py

```diff
@@ -100,31 +110,25 @@ def get_trip_features(source_ids, grid, mode="origin"):
     return {"features": trip_features, "total_trips": total_trips}
 
 
-class Api(Resource):
-    def get(self):
-        args = parser.parse_args()
+@app.get("/v1/trips")
```

Comment:
`v1` 👏

Reply:
just discovered that sanic has built-in methods for versioning. adding an issue for that:
https://sanic.readthedocs.io/en/latest/sanic/versioning.html